src/hotspot/cpu/ppc/stubGenerator_ppc.cpp

```diff
@@ -4331,24 +4275,31 @@ class StubGenerator: public StubCodeGenerator {
     __ lxv(expand_lshift->to_vsr(), 0, const_ptr);
     __ load_const_optimized(const_ptr, (address)&expand_lshift_mask_val, tmp_reg);
     __ lxv(expand_lshift_mask->to_vsr(), 0, const_ptr);
+    __ load_const_optimized(const_ptr, (address)&base64_00_15, tmp_reg);
+    __ lxv(vec_base64_00_15->to_vsr(), 0, const_ptr);
+    __ load_const_optimized(const_ptr, (address)&base64_16_31, tmp_reg);
+    __ lxv(vec_base64_16_31->to_vsr(), 0, const_ptr);
+    __ load_const_optimized(const_ptr, (address)&base64_32_47, tmp_reg);
+    __ lxv(vec_base64_32_47->to_vsr(), 0, const_ptr);
+    __ load_const_optimized(const_ptr, (address)&base64_48_63, tmp_reg);
+    __ lxv(vec_base64_48_63->to_vsr(), 0, const_ptr);
```

Comment:
These two lines are not necessary? The vec_base64_48_63 looks being overwritten in latter lines.